Adds documentation for setting up the IPv6 monitoring VM.

nkinkade opened the pull request.

stephen-soltesz commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[IPV6_MONITORING.md, line 4 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z-9fgpfcmO2ZK7t-y:-L8Z-9fgpfcmO2ZK7t-z:b8yb0vz) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L4)):*
> ```Markdown
> 
> Google Cloud Platform does not currently support IPv6 for most applications.
> Since Prometheus and the Prometheus
> ```

Incomplete sentence? Or extra new line?

---

*[IPV6_MONITORING.md, line 34 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z13lqFsJCkHjTpven:-L8Z13lqFsJCkHjTpveo:b-2gqexd) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L34)):*
> ```Markdown
> 
> * [Enable NDP
>   proxying](https://docs.docker.com/v17.09/engine/userguide/networking/default_network/ipv6/#using-ndp-proxying)
> ```

Is this proxying a consequence of the /116, or would this be required for any subnet prefix?

---

*[IPV6_MONITORING.md, line 64 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z2DYom99gycugfFya:-L8Z2DYom99gycugfFyb:b2isgdq) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L64)):*
> ```Markdown
>   prom/blackbox-exporter`.
> 
> * Manually upload [the blackbox\_exporter config
> ```

These are "first time setup" instructions?

---

*[IPV6_MONITORING.md, line 77 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z2JpDTGFWP-fO6-SJ:-L8Z2JpDTGFWP-fO6-SK:b-mrysdj) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L77)):*
> ```Markdown
> blackbox-exporter-config-mlab-oti.yml
> ```
> * Instantiate the Docker containers (as the user mlab).
> ```

Perhaps emphasize the importance and consequence of the "--restart always" option. If I understand correctly, this follows nicely from the above steps being "first time setup" operations.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-:-L8Z2gxuYfWM8Phu-KBO:bnfp4nl)*
<!-- Sent from Reviewable.io -->

nkinkade commented:
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions, all commit checks successful.

---

*[IPV6_MONITORING.md, line 4 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z-9fgpfcmO2ZK7t-y:-L8Z30O5l6aN2BeKbRxC:bhnnfg4) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L4)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Incomplete sentence? Or extra new line?
</blockquote></details>

Oop. Yes, it was a mistaken linebreak. Fixed.

---

*[IPV6_MONITORING.md, line 34 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z13lqFsJCkHjTpven:-L8Z3MlO5sIJQ7Pg7UyV:b-dzpxit) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L34)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Is this proxying a consequence of the /116, or would this be required for any subnet prefix?
</blockquote></details>

Added a note about this being a consequence of the /116, and that it wouldn't be necessary, apparently, if we had a full /64.

---

*[IPV6_MONITORING.md, line 64 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z2DYom99gycugfFya:-L8Z3b1LsImoW_pSCpjI:bfwda4n) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L64)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

These are "first time setup" instructions?
</blockquote></details>

Yeah, they are more first-time setup instructions, than requirements. I've modified the heading prior to the bullets to point this out.

---

*[IPV6_MONITORING.md, line 77 at r1](https://reviewable.io:443/reviews/m-lab/prometheus-support/204#-L8Z2JpDTGFWP-fO6-SJ:-L8Z47P53_LywOAOICa3:byvwo2v) ([raw file](https://github.com/m-lab/prometheus-support/blob/d9e8b6b328f8a9ee2b7ab576ea1a6f51236ac70b/IPV6_MONITORING.md#L77)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Perhaps emphasize the importance and consequence of the "--restart always" option. If I understand correctly, this follows nicely from the above steps being "first time setup" operations.
</blockquote></details>

Added a note to emphasize that `--restart always` should not be omitted.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/prometheus-support/204)*
<!-- Sent from Reviewable.io -->
